modules/ROOT/pages/install-aws.adoc

```diff
@@ -7,10 +7,10 @@ This topic describes how to install Anypoint Runtime Fabric on your AWS account.
 
 [WARNING]
 ====
-Before installing Runtime Fabric, ensure your infrastructure meets the minimum hardware, operating system, 
-and networking requirements.
+Before installing Runtime Fabric, refer to xref:index.adoc#shared-responsibility-for-on-premises-runtime-fabric[Shared Responsibility for On-Premises Runtime Fabric]. If your infrastructure does not meet the minimum hardware, operating system, 
```

Comment:
Front-load the xref:
```suggestion
Before installing Runtime Fabric, ensure that your infrastructure meets all of the minimum hardware, operating system, and networking requirements by referring to refer to xref:index.adoc#shared-responsibility-for-on-premises-runtime-fabric[Shared Responsibility for On-Premises Runtime Fabric]; otherwise, Runtime Fabric will not operate successfully.
```
OR
```suggestion
Before installing Runtime Fabric, ensure that your infrastructure meets all of the minimum hardware, operating system, and networking requirements (xref:index.adoc#shared-responsibility-for-on-premises-runtime-fabric[Shared Responsibility for On-Premises Runtime Fabric]), or Runtime Fabric will not operate successfully.
```